user-guide/Advanced_Modules/DOM/DOM_examples/DOM_Altering_values_of_a_DomInstance.md

```diff
@@ -0,0 +1,276 @@
+if (existingSection == null)
+{
+    engine.ExitFail($"No Section found on the DomInstance that is linked to the SectionDefinition with ID '{sectionDefinitionId}'");
+    return;
+}
+
+// Remove the FieldValue from the Section.
+existingSection.RemoveFieldValueById(fieldDescriptorId);
+
+// Save the update to the DB.
+// Since we removed the FieldValue on the Section instance retrieved from the DomInstance,
+// we do not need to add the Section again. We have removed it from the reference that is already on the DomInstance.
+domHelper.DomInstances.Update(createdDomInstance);
+```
+
+##### Example: removing a 'Section'
+
+```csharp
+// Find the Section on the DomInstance.
+var existingSection = createdDomInstance.Sections.FirstOrDefault(x => x.SectionDefinitionID.Equals(sectionDefinitionId));
+if (existingSection == null)
+{
+    engine.ExitFail($"No Section found on the DomInstance that is linked to the SectionDefinition with ID '{sectionDefinitionId}'");
+    return;
+}
+
+// Remove the Section from the DomInstance.
+createdDomInstance.Sections.Remove(existingSection);
+
+// Save the update to the DB.
+var updatedInstance = domHelper.DomInstances.Update(createdDomInstance);
+```
+
+##### Example: retrieving a single value
+
+```csharp
+// Find the Section on the DomInstance.
+var existingSection = createdDomInstance.Sections.FirstOrDefault(x => x.SectionDefinitionID.Equals(sectionDefinitionId));
+if (existingSection == null)
+{
+    engine.ExitFail($"No Section found on the DomInstance that is linked to the SectionDefinition with ID '{sectionDefinitionId}'");
+    return;
+}
+
+// Find the FieldValue on the Section.
+var existingFieldValue = existingSection.GetFieldValueById(fieldDescriptorId);
+var valueWrapper = existingFieldValue.Value as ValueWrapper<string>; // Cast to the ValueWrapper of the expected type.
+string actualValue = valueWrapper.Value;
+```
+
+##### Example: retrieving a list value
+
+Just like normal single values, you can add, update, and get list values. Just keep in mind that the `ValueWrapper` should be replaced with `ListValueWrapper`.
+
+```csharp
+// Creating a FieldValue with a list.
+var newFieldValue = new FieldValue(fieldDescriptorId)
+{
+    Value = new ListValueWrapper<string>(new List<string> { "Hello", "Is it me", "You are looking for" } )
+};
+
+// Retrieving a list value.
+var fieldValue = section.GetFieldValueById(fieldDescriptorId);
+var valueWrapper = fieldValue.Value as ListValueWrapper<string>;
+List<string> actualValue = valueWrapper.Values;
+```
+
+## Notes
+
+- Make sure that the value is of the same type as was defined in the associated `FieldDescriptor`. We recommend that you use an IDE that allows you to easily see the type by hovering over it. Deceiving situations might occur, e.g. an 'int' is assigned but the `FieldDescriptor` expects type 'long' (Use '236L' instead of '236' in this case).
```

Comment:
Could you use double quotes instead of single quotes here? To match the Microsoft style, we should use double quotes everywhere except in titles (as markdown does not support them there).